.github/workflows/release.yml

```diff
@@ -0,0 +1,102 @@
+---
+name: 🔖 Release
+
+on:
+  push:
+    tags:
+      - "*"
+
+permissions: {}
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      attestations: write
+      contents: write
+      id-token: write
+      packages: write
+    steps:
+      - name: Checkout
+        id: checkout
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      
+      - name: Set up Docker Context for Buildx
+        id: buildx-context
+        run: docker context use builders || docker context create builders
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@c47758b77c9736f4b2ef4073d4d51994fabfe349 #v3.7.1
+        with:
+          version: latest
+          endpoint: builders
```

Comment:
Why is this required?

Reply:
sorry missed this these arent required I think they will actually break on the public runners ill swap it with the `- name: Install Docker Compose`